Cargo.lock

```diff
@@ -4,17 +4,17 @@ version = "0.0.1"
 dependencies = [
  "byteorder 0.5.1 (registry+https://github.com/rust-lang/crates.io-index)",
  "bytes 0.3.0 (git+https://github.com/carllerche/bytes)",
- "clippy 0.0.65 (registry+https://github.com/rust-lang/crates.io-index)",
+ "clippy 0.0.66 (registry+https://github.com/rust-lang/crates.io-index)",
```

Comment:
why update all dependencies here? Only kvproto is enough. 
You should use another PR to update others.